pkg/arvo/app/publish.hoon

```diff
@@ -127,149 +141,62 @@
-        ==
-      =/  act=invite-action  [%invite /publish uid inv]
-      [%pass /invite %agent [who %invite-hook] %poke %invite-action !>(act)]
+    ++  convert-note-2-3
+      |=  prev=note-2
+      ^-  note-3
+      %=    prev
+          comments
+        [(~(run by comments.prev) convert-comment-2-3) %.n]
+      ==
     ::
-    ++  move-files
-      |=  old-subs=(jug @tas @p)
-      ^-  (list card)
-      =+  ^-  [cards=(list card) sob=soba:clay]
-        %+  roll  .^((list path) %ct (weld our-beak:main /web/publish))
-        |=  [pax=path car=(list card) sob=soba:clay]
-        ^-  [(list card) soba:clay]
-        ?+    pax
-            [car sob]
-        ::
-            [%web %publish @ %publish-info ~]
-          =/  book-name  i.t.t.pax
-          =/  old=old-info  .^(old-info %cx (welp our-beak:main pax))
-          =/  group-pax  /~/(scot %p our.bol)/[book-name]
-          =/  book=notebook-info
-            [title.old '' =(%open comments.old) / /]
-          =+  ^-  [grp-car=(list card) write-pax=path read-pax=path]
-            (make-groups:main book-name [group-pax ~ %.n %.n] title.old '')
-          =.  writers.book      write-pax
-          =.  subscribers.book  read-pax
-          =/  inv-car  (send-invites book-name (~(get ju old-subs) book-name))
-          :-  :(weld car grp-car inv-car)
-          ^-  soba:clay
-          :+  [pax %del ~]
-            :-  /app/publish/notebooks/[book-name]/publish-info
-            [%ins %publish-info !>(book)]
-          sob
-        ::
-            [%web %publish @ @ %udon ~]
-          =/  book  i.t.t.pax
-          =/  note  i.t.t.t.pax
-          :-  car
-          :+  [pax %del ~]
-            :-  /app/publish/notebooks/[book]/[note]/udon
-            [%ins %udon !>(.^(@t %cx (welp our-beak:main pax)))]
-          sob
-        ::
-            [%web %publish @ @ @ %publish-comment ~]
-          =/  book  i.t.t.pax
-          =/  note  i.t.t.t.pax
-          =/  comm  i.t.t.t.t.pax
-          =/  old=old-comment  .^(old-comment %cx (welp our-beak:main pax))
-          =/  new=comment  [creator.old date-created.old content.old]
-          :-  car
-
-          :+  [pax %del ~]
-            :-  /app/publish/notebooks/[book]/[note]/[comm]/publish-comment
-            [%ins %publish-comment !>(new)]
-          sob
-        ==
-      [[%pass /move-files %arvo %c %info q.byk.bol %& sob] cards]
+    ++  convert-notebook-2-3
+      |=  prev=notebook-2
+      ^-  notebook-3
+      %=    prev
+          notes
+        %-  ~(run by notes.prev)
+        |=  =note-2
+        (convert-note-2-3 note-2)
+      ==
     --
   ::
   ++  on-poke
     |=  [mar=mark vas=vase]
     ^-  (quip card _this)
-    ?+  mar  (on-poke:def mar vas)
-        %noun
-      ?:  =(%print-state q.vas)
-        ~&  state
-        [~ this]
-      ?:  =(%print-bowl q.vas)
-        ~&  bol
-        [~ this]
-      [~ this]
+    ?+  mar
+      (on-poke:def mar vas)
```

Comment:
Just a minor style thing, but any reason not to put this in-line? Seems pretty standard

Reply:
I have no opinion